Web/assignment-servelets/Assignment2/WebContent/JavaScript.js

```diff
@@ -0,0 +1,35 @@
+function activateSubmit() 
+{
+	if(document.getElementById("acceptCondition").checked == true)
+	{
+		document.getElementById("button").disabled = false;
```

Comment:
ACTION: Alignment should be proper and use proper name for js file instead of Javascript.js